manuscript/03.results.md

```diff
@@ -4,29 +4,29 @@
+Given the unique insights that single cell measurements on lineage trees can provide, we implemented a strategy for classifying cells based on their phenotype and lineage relationships. Given a set of measurements made across a lineage tree, we used a lineage tree-based hidden Markov model (tHMM) to fit these data (Fig. {@fig:tree}a). Like a typical hidden Markov model, a tHMM can infer the hidden discrete "states" of cells given a series of measurements where a state is defined by specific phenotype distributions. The inference of these states takes place using an iterative strategy wherein the states of each cell are predicted by the phenotype of both the cell and its relatives in a lineage ("expectation" step), and then each distribution of phenotypes is fit to match the cells within that state ("maximization" step) (Fig. {@fig:tree}b). This expectation-maximization (EM) process repeats until convergence.
 
-After fitting, the model can provide a variety of information (Fig. {@fig:tree}c). First, it estimates the starting abundance of each state and the transition probabilities between them. Second, the distribution of cells' phenotypes in each state are estimated and can be compared to distinguish how cells of each state behave. The state of each cell can be predicted from the fit data or new measurements. Finally, the model provides a likelihood of each cell’s observations and therefore the data overall. This last quantity can be used, for example, to estimate the number of distinguishable cell states. When implementing these processes, we ensured that a cell’s measurements are defined through a modular interface, allowing other forms of data to be easily integrated.
+After fitting, the model can provide a variety of information (Fig. {@fig:tree}c). First, it estimates the abundance of each state at time = 0 and the transition probabilities between these states which are the hyper-parameters of our tHMM model. Second, the distribution of cells' phenotypes in each state are estimated and can be compared to distinguish how cells of each state behave. The state of each individual cell can be predicted from the fit data or new measurements. Finally, the model provides a likelihood of each cell’s observations and therefore the data overall. This last quantity can be used, for example, to estimate the number of distinguishable cell states. When implementing these processes, we ensured that a cell’s measurements are defined through a modular interface, allowing other forms of data to be easily integrated.
```

Comment:
Isn't "starting" more natural than "t=0"?

Reply:
yes, but Dr. Gross asked the difference between starting abundance and the distribution, I thought this makes it a little bit more clear. I will change the sentence to emphasize that.